cloudevents/conversion.py

```diff
@@ -23,15 +23,44 @@
 #    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 #    License for the specific language governing permissions and limitations
 #    under the License.
+import datetime
+import enum
 import json
 import typing
 
 from cloudevents import exceptions as cloud_exceptions
 from cloudevents.abstract import AnyCloudEvent
-from cloudevents.http import is_binary
-from cloudevents.http.mappings import _marshaller_by_format, _obj_by_version
-from cloudevents.http.util import _json_or_string
 from cloudevents.sdk import converters, marshaller, types
+from cloudevents.sdk.converters.binary import is_binary
+from cloudevents.sdk.event import v1, v03
+
+_AnyT = typing.TypeVar("_AnyT", bound=typing.Any)
+
+
+def _best_effort_serialize_to_json(
+    value: _AnyT, *args, **kwargs
+) -> typing.Optional[typing.Union[bytes, str, _AnyT]]:
```

Comment:
How `Any` is different from `AnyT`? `Any` is already whatever you want :-) I don't think it worth having a generic in this case.